style.css

```diff
@@ -10,3 +10,52 @@ body {
   background: #344a72;
   color: #fff;
 }
+
+#header {
+  display: flex;
+  justify-content: space-between;
+}
+
+#header ul {
+  display: flex;
+  list-style: none;
+  margin: 20px;
+}
+
+#logo {
+  font-family: 'Whisper', cursive;
+  font-size: 40px;
+  cursor: pointer;
+}
+
+li {
+  padding-right: 10px;
+}
+
+a {
+  color: pink;
+}
+
+a:visited {
+  color: pink;
+  text-decoration: none;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: white;
+  color: black;
+  max-width: 300px;
+  height: 300px;
+  margin-left: auto;
+  margin-right: auto;
+  border-radius: 10px;
+  margin-top: 100px;
+}
+
+h1 {
+  color: brown;
+}
```

Comment:
- [ ] You are using an element selector to style the heading, Please use a class selector instead to follow the project requirements. You can add a class to the heading and use that class in the .css file to style the heading.

Reply:
Thanks for the review. I'll make the appropriate changes.